### DISCUSSION TITLE
Display Last Full Scan time

### DISCUSSION THREAD

**Discussion by dathbe** [navidrome/navidrome#2817] [Ideas]
I would like the UI to display (perhaps only for admins) the timestamp for the last full scan.  This would let us know how recently the folders were scanned and whether a new scan needs to be initiated.  It would fit nicely in this pop-up:

![2024-01-25 14_16_34-Navidrome — LibreWolf](https://github.com/navidrome/navidrome/assets/14321825/931ede61-7197-421d-9c98-16c3ad82379f)

**Comment by deluan** [comment]
That's actually a good idea! But this will have to wait for the new UI, and this pop-up will probably change and also include things like who is listening to what (just admins, ofc)
